### ISSUE TITLE
CodeMirror stopped working after version 6.0.14. Why minor version contains breaking changes?

### ISSUE THREAD

**Issue by roko91** [JeffreyWay/laravel-mix#2927]
-   Laravel Mix Version: 6.0.16
-   Node Version (`node -v`): v14.16.0
-   NPM Version (`npm -v`): 6.14.11
-   OS: Ubuntu 20.04

### Description:
After version 6.0.14 both CodeMirror and jquery-toast-plugin both have broken styles which rendered them useless. I'm still not sure what the issue is, since there is no errors and the end bundle size seems about the same but it may have something to do with importing CSS files in JS package.

Genuine question though: why minor version contains breaking changes?
It took me almost an hour to narrow it down to Laravel Mix and that was the last package I expected to cause an issue since it only came up from 6.0.11 to 6.0.16... I though semantic versioning was created to avoid such situations. Thank you for your hard work but I'm a bit frustrated by that.
Now I just want to know what to change in order to move forward and update because now I'm stuck on 6.0.11 until this is resolved.

### Steps To Reproduce:
Install CodeMirror or jquery-toast-plugin and import their styles in the JavaScript file:
`import CodeMirror from 'codemirror';
import 'codemirror/lib/codemirror.css';
import 'codemirror/theme/material.css';
import 'codemirror/mode/css/css';
import 'codemirror/mode/javascript/javascript';
import 'codemirror/addon/display/autorefresh';`
or
`import 'jquery-toast-plugin/dist/jquery.toast.min.css';
import 'jquery-toast-plugin';`

**Comment by thecrypticace** [comment]
The apparent breaking change is *not* on purpose. We have tests for these kinds of things and they all pass currently. I'll try to take a look tonight to see if I can figure out what's wrong here.

**Comment by roko91** [comment]
Thank you for the quick reaction.
Sorry if my tone was a bit harsh... it's just that when I downgraded I saw 17 packages removed, 24 added (or something similar) and I said to myself "this looks like a huge update for minor release" and that frustrated me a bit.
I would gladly help, if needed, to get to the bottom of this.

**Comment by thecrypticace** [comment]
So I just tried this and, at least in isolation, the styles are injected into the <head> tag just as they were before. Could you provide a reproduction repository?

**Comment by roko91** [comment]
The project I develop is private but you can look the .vue component I attached.
[AssetEditor.zip](https://github.com/JeffreyWay/laravel-mix/files/6277355/AssetEditor.zip)

**Comment by thecrypticace** [comment]
<img width="314" alt="Screen Shot 2021-04-08 at 08 18 45" src="https://user-images.githubusercontent.com/614993/114025393-0b046400-9843-11eb-9a8c-a09b9b0449a3.png">

Yeah this works fine for me. There does appear to be an issue in Safari where the code doesn't show up but it's only at random times (… which is concerning tbh but not a Mix issue) and does not happen in Chrome. And this was regardless of mix version.

The only thing I can think of is that it's related to your mix configuration or some other dependencies you have installed — could you reproduce this issue in a fresh project?

**Comment by roko91** [comment]
Sorry for the delayed response. I just checked on fresh Laravel project today and everything is working fine... maybe there is some conflict with something else I'm using. I'll keep investigating.

**Comment by roko91** [comment]
I stripped my project to make it absolutely identical with the fresh one and styles still didn't load... then I decided to diff the two `app.js` bundle files and turns out the fresh project has this line:
`if(content.__esModule) content = content.default;`
while my project bundle doesn't, and that's the only difference.
I have no idea how is that possible considering everything is the same.
I even copied the same `package.json`, deleted `node_modules`, did run `npm install again` and still this line is missing :(

**Comment by thecrypticace** [comment]
Can you post a gist of the package-lock.json file from the fresh project and the stripped down project?

**Comment by roko91** [comment]
As I said I copied them to be absolutely the same, same goes for `app.js` and `webpack.mix.js` but here it is:
https://gist.github.com/roko91/c75aa9078cbf938bc551450bf3c36bef

**Comment by thecrypticace** [comment]
Yes but not the `package.json`. I wanted to look at `package-lock.json`.

**Comment by roko91** [comment]
Oh, sorry, I misread. I completely forgot about the lock file when trying to replicate... now I deleted it as well as the `node_modules` folder, then did run `npm install` again but before that I added `postcss-import` in `package.json` 'cuz I noticed it was missing and viola - everything is working fine now! :smile: 
Thank you a lot for the help, I suspect it was the missing `postcss-import` which isn't dependency for some reason.
